lib/at_cmd_parser/at_params.c

```diff
@@ -10,38 +10,36 @@
 #include <string.h>
 #include <zephyr.h>
 #include <zephyr/types.h>
-#include <at_params.h>
 #include <kernel.h>
 
-/* Internal function. */
+#include <at_cmd_parser/at_params.h>
+
+/* Internal function. Parameter cannot be null. */
 static void at_param_init(struct at_param *param)
 {
-	__ASSERT(param != NULL, "Parameter pointer cannot be NULL.\n");
+	__ASSERT(param != NULL, "Parameter cannot be NULL.");
 
+	/* Initialize to default. Empty parameter with '0'/null value. */
```

Comment:
`Empty parameter with '0'/null value.` - I think it should be either `'\0'` or `NULL` - it's the same, no need to repeat.

Reply:
Ok, changed to only null